x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/alerts_page.tsx

```diff
@@ -205,6 +205,7 @@ function InternalAlertsPage() {
                 featureIds={observabilityAlertFeatureIds}
                 query={esQuery}
                 showExpandToDetails={false}
+                showAlertStatusWithFlapping={true}
```

Comment:
nit: For true boolean params, we don't need to specify the `true` value

```suggestion
                showAlertStatusWithFlapping
```